app/views/users/show.html.erb

```diff
@@ -3,7 +3,7 @@
 <p>Email: <%= @user.email %></p>
 <p>Console: <%= @user.console_name %></p>
 <p>Team Name: <%= @user.team_name %></p>
-<p>Reputation: <%= @user.reputation.nil? ? "No trades yet" : @user.reputation %></p>
```

Comment:
It might be better practice here to use `#blank?` instead of `#nil?` since the former checks against empty strings as well.